2025-08-12 20:14:35,847 - ERROR - rexus.errors - ERROR
Message: Uncaught exception: UnicodeEncodeError: 'charmap' codec can't encode character '\u2713' in position 0: character maps to <undefined>
File: C:\Users\itachi\Desktop\martin\Rexus.app\rexus\utils\error_handler.py:24
Function: handle_exception
---
Traceback (most recent call last):
  File "<string>", line 17, in <module>
  File "C:\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2713' in position 0: character maps to <undefined>
2025-08-14 07:51:12,614 - ERROR - rexus.errors - ERROR
Message: Uncaught exception: UnicodeEncodeError: 'charmap' codec can't encode character '\u2705' in position 12: character maps to <undefined>
File: C:\Users\itachi\Desktop\martin\Rexus.app\rexus\utils\error_handler.py:24
Function: handle_exception
---
Traceback (most recent call last):
  File "<string>", line 12, in <module>
  File "C:\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2705' in position 12: character maps to <undefined>
2025-08-14 12:49:51,303 - ERROR - rexus.errors - ERROR
Message: Uncaught exception: UnicodeEncodeError: 'charmap' codec can't encode character '\U0001f527' in position 0: character maps to <undefined>
File: C:\Users\itachi\Desktop\martin\Rexus.app\rexus\utils\error_handler.py:22
Function: handle_exception
---
Traceback (most recent call last):
  File "<string>", line 44, in <module>
  File "C:\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\U0001f527' in position 0: character maps to <undefined>
2025-08-14 12:49:51 - rexus.errors - [31mERROR[0m - Uncaught exception: UnicodeEncodeError: 'charmap' codec can't encode character '\U0001f527' in position 0: character maps to <undefined>
Traceback (most recent call last):
  File "<string>", line 44, in <module>
  File "C:\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\U0001f527' in position 0: character maps to <undefined>
2025-08-15 12:55:35,037 - ERROR - rexus.errors - ERROR
Message: Uncaught exception: UnicodeEncodeError: 'charmap' codec can't encode character '\u2713' in position 0: character maps to <undefined>
File: C:\Users\itachi\Desktop\martin\Rexus.app\rexus\utils\error_handler.py:22
Function: handle_exception
---
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2713' in position 0: character maps to <undefined>
2025-08-15 12:55:35 | ERROR    | rexus.errors    | Uncaught exception: UnicodeEncodeError: 'charmap' codec can't encode character '\u2713' in position 0: character maps to <undefined>
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2713' in position 0: character maps to <undefined>
